{{ cookiecutter.project_name }}/tox.ini

```diff
@@ -8,6 +8,7 @@ setenv =
     SECRET_KEY='FOO'
     ALLOWED_HOSTS=localhost
     DATABASE_URL=postgres://localhost/{{ cookiecutter.project_db }}
+    SECURE_SSL_REDIRECT=False
```

Comment:
You might not always run tests with tox. It's too slow. WHen hacking you probably want to run a specific `./manage.py test ....` command. So, if the tests break if `SECURE_SSL_REDIRECT` we ought to solve it generally. Probably in `settings.py`.

On a django 1.10 side-project I have, I do this:
```python
# This is the bottom of settings.py
if 'test' in sys.argv[1:2]:
    os.environ['OPBEAT_DISABLE_SEND'] = 'true'
    CACHES = {
        'default': {
            'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
            'LOCATION': 'unique-snowflake',
        }
    }

    DOWNLOADER_API_KEYS = {'deadbeef33': 'peterbe'}
```